make/Makerules

```diff
@@ -128,6 +128,10 @@ endef
 #   @return C-style string with each character as a raw hex byte ("\xNN\xNN...\xNN")
 uuidToCstr = "$(shell echo $(subst -,,$(1)) | $(SED) 's/.\{2\}/\\x&/g')"
 
+# Convert AES-KEY string to a C representation of raw bytes
+#   @param $(1) AES-KEY 16 bytes (nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn)
+#   @return C-style string with each character as a raw hex byte ("\xNN\xNN...\xNN")
+aesToCstr = "$(shell echo $(1) | $(SED) 's/.\{2\}/\\x&/g')"
```

Comment:
This is a general purpose function for converting any set of bytes properly presented, not only AES keys. Perhaps name it so?

Reply:
Fixed in commit 1c97ae3e08a7b26ed4601fe9774ebfd9040941fe